lib/active_merchant/billing/gateways/paypal/paypal_common_api.rb

```diff
@@ -637,11 +650,16 @@ def commit(action, request)
           :test => test?,
           :authorization => authorization_from(response),
           :fraud_review => fraud_review?(response),
+          :error_code => error_code(response),
           :avs_result => { :code => response[:avs_code] },
           :cvv_result => response[:cvv2_code]
         )
       end
 
+      def error_code(response)
+        STANDARD_ERROR_CODE_MAPPING[response[:error_codes]] || response[:error_codes]
```

Comment:
`error_code` should only return a standardized error code. Instead of returning Paypal's original code, you could default to `processing_error`